lib/internal/Magento/Framework/GraphQlSchemaStitching/GraphQlReader/TypeMetaReaderInterface.php

```diff
@@ -18,5 +20,5 @@ interface TypeMetaReaderInterface
      * @param \GraphQL\Type\Definition\Type $typeMeta
      * @return array|null
      */
-    public function read(\GraphQL\Type\Definition\Type $typeMeta)  : array;
+    public function read(\GraphQL\Type\Definition\Type $typeMeta): ?array;
```

Comment:
Please, do not change the return type of this function. This is backward incompatible and may create other problems.